src/vs/workbench/common/actions.ts

```diff
@@ -110,7 +110,11 @@ Registry.add(Extensions.WorkbenchActions, new class implements IWorkbenchActionR
 
 				const from = args && args.from || 'keybinding';
 
-				return Promise.resolve(actionInstance.run(undefined, { from })).then(() => {
+				if (args) {
```

Comment:
@RMacfarlane the change looks good to me except for this piece that scares me. Are we sure we are not sending arbitrary data to all actions that get executed that might expect something else here? There is only one action where we need this for (Start Debug) but for any action that is triggered we now pass the args on. This has the potential of breaking badly.

Reply:
Yeah, this was the code I wasn't sure about. Since only Start Debug needs the args passed on, I reverted this change and added separate code for registering that command